kubeflow-pipelines.yaml

```diff
@@ -213,7 +213,7 @@ subpackages:
     pipeline:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}/kfp/metadata_writer
-          ln -s /usr/lib/python3.11/site-packages/metadata_writer.py "${{targets.subpkgdir}}"/kfp/metadata_writer/
+          ln -s /usr/lib/python3.12/site-packages/metadata_writer.py "${{targets.subpkgdir}}"/kfp/metadata_writer/
```

Comment:
Could we use a wildcard so when we've a minor python3.x bump again in future we'll not need to increment this?

```
ln -s /usr/lib/python3.*/site-packages/metadata_writer.py "${{targets.subpkgdir}}"/kfp/metadata_writer/
```

Haven't tested but assume that'd work?

Reply:
let me quickly check that! great idea.

Reply:
unfortunately, it didn't work.

Reply:
or you could get version from python :)

```sh
$ python -c 'import sys; print("%s.%s" % (sys.version_info.major, sys.version_info.minor))'
3.11
```